are_we/tools/off.py

```diff
@@ -0,0 +1,10 @@
+import PyCmdMessenger
+
+port = '/dev/ttyACM0'
+ard = PyCmdMessenger.ArduinoBoard(port, baud_rate=115200)
```

Comment:
Consider making the baud_rate a global constant so that in future development as the code expands we can change the baud rate more easily if needed.

Reply:
^ +1